components/common/FormSection.tsx

```diff
@@ -0,0 +1,38 @@
+import { Box, Flex } from "@chakra-ui/react"
+import React from "react"
+import ErrorAlert from "./ErrorAlert"
+import { SubHeader } from "./Header"
+
+interface IFormSectionProps {
+    header: string
+    errorMessage?: string
+    children: JSX.Element | Array<JSX.Element>
+}
+
+const FormSection = (props: IFormSectionProps): JSX.Element => {
+    const { header, children, errorMessage } = props
+    return (
+        <>
+            <Box paddingX="2rem" paddingY="1rem">
+                <SubHeader text={header} />
+                <Flex
```

Comment:
Perhaps related to @LeadingMoominExpert comments, I would prefer us to find a way to separate styles from render statements such as this one right here. I really like the way things were done in my previous project, [see e.g. this](https://github.com/espoon-voltti/evaka/blob/master/frontend/src/employee-frontend/components/finance-basics/FeeThresholdsItem.tsx#L205-L222). I think something that looks like that would promote recycling styles and avoid repetition.

Might be too much work to tackle in this PR though.

Reply:
Not in scope of this PR